extensions/vertx-web/runtime/src/main/java/io/quarkus/vertx/web/runtime/VertxWebRecorder.java

```diff
@@ -84,6 +85,12 @@ public void addAdditionalRoutes(RuntimeValue<Router> routerRuntimeValue, Map<Str
         if (maxBodySize.isPresent()) {
             bodyHandler.setBodyLimit(maxBodySize.get().asLongValue());
         }
+        final BodyHandlerConfig bodyHandlerConfig = httpConfiguration.bodyHandler;
```

Comment:
Should it be just "body"?
For example, when we will have the equivalent for the `cacheHandler`, or for `staticFileHandler`, we will _hide_ the handler part. WDYT?

Reply:
+1 let's remove the handler and also the reference to Wert.x web in the JavaDoc.

Reply:
Addressed in 950b2ad4e552f342f0a4adba99fe99a4ac5bc8de